core/src/main/java/nu/yona/server/analysis/entities/DayActivity.java

```diff
@@ -44,6 +45,7 @@
 							// so you get 4 goals * 1 day = 4 activity collections to be joined
 	private List<Activity> activities;
 
+	@Column(columnDefinition = "bit default false")
```

Comment:
Why are those lines exactly necessary? Is that a still remaining bug in Liquibase? Because a boolean is false by default in Java, so I would expect that it is saved as false to the database.

Reply:
I agree it's a bit weird, but I wouldn't call it a bug. In SQL, a column can be null. In some scenarios (because of our mapping strategy), we do not use this column. In such cases, we need to tell what the column value should be, as otherwise it cannot be read back in a boolean.

Reply:
I still don't get why it shouldn't be simply `null` in the case that we do not use the column because of our mapping strategy. In that case it should also not be necessary to read it back to a boolean, should it?

Reply:
Indeed, I had similar questions. We only read it if we write it, so a default value shouldn't matter.